LICENSE

```diff
@@ -7,7 +7,7 @@ Licensed Work:        Omni
                       The Licensed Work is (c) 2024 Sidero Labs, Inc.
 Additional Use Grant: None
 
-Change Date:          2028-08-16
+Change Date:          2028-02-22
```

Comment:
why did it go back in time?

Reply:
Oh, this is kres's doing, but I don't know why. Let me have a look.

Reply:
It is because it takes the `ChangeDate` template param as-is from `.kres.yaml` when rekres-ed. I updated it. Maybe we should make kres smarter about it somehow, if you have a good idea. For example, we could leave out `ChangeDate` to be a param, and always set it to be `today+4y` or something like that.

Reply:
the whole idea was to bump `Change Date` to be `now` + N years each time you do a release on `main`.

Reply:
otherwise this is basically no-op

Reply:
it still works, but only when you rekres, it takes it back unless you go and update `.kres.yaml`. that is the problem. when we don't rekres, it works as expected.

Reply:
then it should actually update `.kres.yaml`? idk

but the goal I stated above, the implementation can be anything

Reply:
yes, that would work, just did that here: https://github.com/siderolabs/kres/pull/437